<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmPassword</class>
 <widget class="QDialog" name="frmPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TopoDelProp: Usuarios.</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuario:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtUsuario">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPsw1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPsw2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Contraseña:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirme contraseña:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bttAceptar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bttCancelar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Condiciones para crear usuarios</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Asegúrese de:
       - Que la longitud del nombre de usuario está entre 10 y 25 caractéres.
        - Que comienza por una letra del abecedario. Esto es necesario para los
            usuarios de postgres.
        - Que la contraseña tiene una longitud mínima de 15 caractéres y:
            - Tiene algún número.
            - Tiene alguno de los siguientes caractéres =,&lt;,&gt;,@,#,%,$</string>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
